### PR TITLE
{data}[intel/2017a] destiny 2.5.6 w/ R 3.4.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2498.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/863c9766938db1964d2eb8413c96386f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2498.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/b5104071e3588b2d928bf63db27f2471 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2140.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/af18b3aced81aa3f083f6a3fe57535e8 for a full test report.

File: easybuild/easyconfigs/d/destiny/destiny-2.5.6-intel-2017a-R-3.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    ('R', '3.4.0', '-X11-20170314'),
+    ('R-bundle-Bioconductor', '3.5', versionsuffix),
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': [name],
+}
+
+moduleclass = 'data'
```
Comment (verdurin) [inline easybuild/easyconfigs/d/destiny/destiny-2.5.6-intel-2017a-R-3.4.0.eb]: Hmm - is this the right `moduleclass`? Isn't everything in `Bioconductor` by definition `bio`?

Comment (boegel) [inline easybuild/easyconfigs/d/destiny/destiny-2.5.6-intel-2017a-R-3.4.0.eb]: `R packages to create and plot diffusion maps.`

To me, this fits `data` (or maybe `vis`) better than `bio`?

Comment (verdurin) [inline easybuild/easyconfigs/d/destiny/destiny-2.5.6-intel-2017a-R-3.4.0.eb]: It imports `Biocbase` and `BiocGenerics` and is included in the `CellBasedAssays` and `CellBiology` `BiocViews`, which sounds pretty `bio` to me.

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**